wiki/BanchoBot/id.md

```diff
@@ -4,29 +4,29 @@ tags:
+  - perintah
+  - pengumuman
 ---
 
 # BanchoBot
 
-*Berbeda konteks dengan [Bancho (server)](/wiki/Bancho_(server))*
+*Untuk penggunaan lainnya, lihat [Bancho (disambiguasi)](/wiki/Disambiguation/Bancho).*
 
 ![Kartu pemain BanchoBot](img/BanchoBot.jpg "Kartu pemain BanchoBot")
 
-**BanchoBot** (atau biasa dikenal dengan *Bancho*) adalah sebuah bot yang dibuat untuk osu! membantu para pemain dalam obrolan dengan menampilkan informasi tertentu dan memberitahukan pesan terkait gim (contoh. total bermain, jumlah pengulangan, dan lain-lain.), dan menanggapi beberapa perintah tertentu. Yang diprogram oleh [Echo](https://osu.ppy.sh/users/431) dan penyedia dari Bancho IRC (Internet Relay Chat).
+**BanchoBot** (atau biasa dikenal dengan *Bancho*) adalah sebuah bot yang dibuat untuk osu! membantu para pemain dalam obrolan dengan menampilkan informasi tertentu dan memberitahukan pesan terkait gim (contoh. total bermain, jumlah pengulangan, dan lain-lain.), dan menanggapi beberapa perintah tertentu. BanchoBot diprogram oleh [Echo](https://osu.ppy.sh/users/431) dan merupakan penyedia dari [IRC Bancho](/wiki/Internet_Relay_Chat) (Internet Relay Chat).
 
 BanchoBot juga memiliki [profil osu!](https://osu.ppy.sh/users/3) dan [akun Twitter](https://twitter.com/banchoboat).
 
-## Perintah BanchoBot
+## Perintah
 
-*Untuk daftar perintah yang menggunakan slash (klien), kunjungi artikel [Konsol Obrolan](/wiki/Chat_Console#daftar-perintah).*
+*Untuk melihat daftar perintah saat berada di dalam gim, kunjungi [Chat Console](/wiki/Chat_Console#daftar-perintah)*
 
-Untuk menggunakan perintah BanchoBot, kamu harus menggunakan awalan `!` setiap saat akan memulai perintah dan setelahnya diikuti dengan nama perintah menggunakan awal kapital (tanpa spasi didalamnya). Perintah ini dapat digunakan di dalam kanal obrolan dan obrolan privat dengan BanchotBot.
+BanchoBot dapat merespon perintah pemain melalui pesan yang diketik secara khusus dalam obrolan. Semua perintah BanchoBot dimulai dengan sebuah tanda seru (`!`) diikuti dengan nama perintah yang tidak mempengaruhi penggunaan huruf besar/kecil (tanpa spasi didalamnya). Perintah ini dapat digunakan di dalam kanal obrolan dan obrolan privat dengan BanchotBot.
 
-Jika pengguna mencoba mengirim perintah dalam obrolan publik, pengguna biasa tidak dapat melihatnya dan pemain yang mengirim perintah akan mendapat jawabannya sebagai pesan pribadi. Pengguna juga dapat menggunakan perintah `/bb <command>` untuk membuka tab BanchoBot secara otomatis dan mengirim perintah dengan segera.
+Jika seorang pengguna biasa mencoba mengirim perintah dalam obrolan publik, pengguna lainnya tidak dapat melihatnya dan pemain yang mengirim perintah akan mendapatkan jawabannya melalui pesan pribadi dari BanchoBot. Pengguna juga dapat menggunakan perintah `/bb` untuk membuka tab BanchoBot secara otomatis dan mengirim perintah dengan segera.
 
-Daftar perintah BanchoBot:
+Daftar keseluruhan dari semua perintah BanchoBot dapat ditemukan dibawah ini:
```

Comment:
`keseluruhan dari semua` feels really awkward here i guess since you're indirectly using "all" twice in a row in a single sentence. i'd go for sth simpler like `Daftar perintah BanchoBot selengkapnya adalah sebagai berikut:`